### PR TITLE
[IMP] point_of_sale, *: revamp UI

### PR DESCRIPTION
**Pull request by xlu-odoo**
*: pos_discount, pos_hr, pos_loyalty, pos_restaurant, pos_sale

This PR implements a comprehensive redesign of the POS interface, featuring substantial enhancements in usability and aesthetics.

The main objective is to improve the overall user experience and deliver a visually new appealing interface.

task-3954669

ENT: https://github.com/odoo/enterprise/pull/68104

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/static/src/app/generic_components/list_container/list_container.js
Comment (caburj) [inline addons/point_of_sale/static/src/app/generic_components/list_container/list_container.js:1]: May I know why we need to make this change?

**Review by caburj — APPROVED** [review]
robodoo r+

**Comment by caburj** [comment]
robodoo r+

File: addons/point_of_sale/static/src/app/navbar/navbar.xml [hunk 1]
```diff
@@ -2,81 +2,88 @@
...
+                    class="'p-0'"
+                    isSmall="ui.isSmall"
+                    placeholder.translate="Search products..."
+                    icon="{type: 'fa', value: 'fa-search fa-lg fa-fw'}"
+                    debounceMillis="500"
+                    t-if="pos.showSearchButton()" />
+                    <button class="btn btn-light btn-lg lh-lg" t-if="isBarcodeScannerSupported() and pos.mainScreen.component.name == 'ProductScreen'" t-on-click="onClickScan">
+                        <i class="fa fa-fw fa-lg fa-barcode" /><span t-if="this.pos.scanning" class="ms-1">Stop</span>
+                    </button>
+                    <div t-if="this.pos.data.network.offline or this.pos.data.network.loading" class="oe_status btn btn-light btn-lg lh-lg h-100 pe-none" t-on-click="onSyncNotificationClick">
```
Comment (rmbh-odoo) [inline addons/point_of_sale/static/src/app/navbar/navbar.xml:37]: `pe-none` is disabling pointer events, which prevents `t-on-click` from working.

**Pull request closed without merging**